src/pip/_internal/commands/install.py

```diff
@@ -616,11 +616,13 @@ def decide_user_install(
     If use_user_site is None, the default behaviour depends on the environment,
     which is provided by the other arguments.
     """
-    if use_user_site is False:
+    # In some cases (config from tox), use_user_site can be set to an integer
+    # rather than a bool. Comparing == True/False instead of 'is' allows this.
+    if use_user_site == False:  # noqa: E712
```

Comment:
I would use `not use_user_site` and drop the noqa and comment.

Reply:
@chrahunt this is a tri-state variable: https://github.com/pypa/pip/pull/7002#issuecomment-545303601